src/test/resources/application-integrasjonstest.yaml

```diff
@@ -146,7 +146,7 @@ AZURE_APP_WELL_KNOWN_URL: http://localhost:${mock-oauth2-server.port}/azuread/.w
 AZURE_OPENID_CONFIG_TOKEN_ENDPOINT: https://login.microsoftonline.com/navq.onmicrosoft.com/oauth2/v2.0/token
 
 FAMILIE_INTEGRASJONER_SCOPE: "dummy"
-SANITY_DATASET: "ks-test"
+SANITY_DATASET: "ks-brev"
```

Comment:
Betyr det at integrasjontestene våre er avhengig av Sanity?

Reply:
Jepp, det stemmer. Burde egentlig bruke sanity.json fil som vi bruker til cucumber testene, men tenker det kan fikses i en annen PR.

Reply:
Fint hvis man lager et kort på dette. Vi bør ikke være avhengig av eksterne tjenester for å bygge eller deploye